pkg/cfgstruct/bind.go

```diff
@@ -20,34 +20,36 @@ import (
 )
 
 // BindOpt is an option for the Bind method
```

Comment:
this is a terrible comment. this explains nothing. what good is this. let's fix it. can you explain more?

Reply:
this is so rude of you